package.json

```diff
@@ -3,6 +3,8 @@
   "version": "3.0.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/lab": "^4.0.0-alpha.57",
```

Comment:
We shouldn't be adding material ui to the ref-store. It will definitely increase the bundle size and the loading times. 
I see that you are using it only for the Skeleton component. You can probably implement this with pure css.

Reply:
Ok, I will change this.